model/registry/rpc.yaml

```diff
@@ -262,3 +262,19 @@ groups:
         type: int
         stability: experimental
         brief: "Uncompressed size of the message in bytes."
+      - id: rpc.request.body.size
+        type: int
+        stability: experimental
+        brief: "The size of the request payload body in bytes."
+        note: >
+          For gRPC, this value is calculated through [protobuf](https://protobuf.dev/reference/java/api-docs/com/google/protobuf/MessageLite.html#getSerializedSize--)
```

Comment:
Thank you for providing the link to the docs!

If I understand correctly, this is a size of the message.

I.e. for unary calls, that's the request size, but for streaming calls, it'll be the size of the individual message and it could be useful on events describing individual streaming messages.

It would be misleading to use request/response size attributes for streaming calls on such message events.  

So I think we have a couple of options:

1. Try using something like `rpc.message.body.size` - it'll look good on events, but won't work for rpc spans (we need to report request and response sizes)
2. Keeping `rpc.request|response.body.size` for the request/response part only.
3. Do both (1+2)

It seems we're doing option 2 which is fine with me, we just need to clarify it further - I'll leave some suggestions.

Reply:
maybe:

```suggestion
          The request body size represents the serialized size of the message payload that is sent over the wire. In case of streaming calls, it accounts for the original message in the request and does not include size of the consequent message stream.
```

Reply:
Thank you very much for your addition👍.